.drone.yml

```diff
@@ -3,143 +3,552 @@ kind: pipeline
 name: default
 type: kubernetes
 
-steps:
-- name: build_are
-  pull: if-not-exists
-  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
-  commands:
-    - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
-    - docker build -t app-$${DRONE_COMMIT_SHA} .
-  when:
-    branch: [ master, feature/* ]
-    event: push
+environment:
+  APP_NAME: appeal-rights-exhausted
+  PROD_ENV: are
+  STG_ENV: are-preprod
+  UAT_ENV: are-dev
+  BRANCH_ENV: are-branch
+  PRODUCTION_URL: www.appeal-rights-exhausted.homeoffice.gov.uk
+  IMAGE_URL: quay.io/ukhomeofficedigital
+  IMAGE_REPO: are-app
+  GIT_REPO: UKHomeOffice/AppealRightsExhausted
+  HOF_CONFIG: hof-services-config/Appeal_Rights_Exhausted
+  NON_PROD_AVAILABILITY: Mon-Fri 08:00-23:00 Europe/London
+  READY_FOR_TEST_DELAY: 20s
+  NOTIFY_STUB: stub
 
-- name: build_acceptance
-  pull: if-not-exists
-  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
-  commands:
-    - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
-    - docker build -f Dockerfile-acceptance -t acceptance-$${DRONE_COMMIT_SHA} .
-  when:
-    branch: [ master, feature/* ]
-    event: push
+trigger:
+  branch:
+    - feature/*
+    - master
 
-- name: test_acceptance
-  pull: if-not-exists
-  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
-  commands:
-  - n=0; while [ "$n" -lt 60 ] && [ ! docker stats --no-stream ]; do n=$(( n + 1 )); sleep 1; done
-  - docker run -d --name=redis-$${DRONE_COMMIT_SHA} redis
-  - docker run -d -e NODE_ENV=ci -e DISABLE_CSP=true --name=app-$${DRONE_COMMIT_SHA} --net=container:redis-$${DRONE_COMMIT_SHA} app-$${DRONE_COMMIT_SHA}
-  - docker run --rm --net=container:app-$${DRONE_COMMIT_SHA} acceptance-$${DRONE_COMMIT_SHA}
-  - docker rm -vf "app-$${DRONE_COMMIT_SHA}" "redis-$${DRONE_COMMIT_SHA}"
-  when:
-    branch: [ master, feature/* ]
-    event: push
-
-- name: image_to_quay
+linting: &linting
   pull: if-not-exists
-  image: 340268328991.dkr.ecr.eu-west-2.amazonaws.com/acp/dind
+  image: node:14
```

Comment:
Is there a reason in this scenario not to use a specific node image with a sha like in the dockerfiles

Reply:
not particularly, it's just for running tests but we could do. It isn't used by any images so less relevant

Reply:
fair enough